client/ide/lib/index.coffee

```diff
@@ -258,6 +258,14 @@ class IDEAppController extends AppController
 
   setActiveTabView: (tabView) ->
 
+    activePane = tabView.getActivePane()
+
+    for view in @ideViews
+      for tab in view.holderView.tabs.subViews
+        if tab is activePane?.tabHandle
+        then activePane.tabHandle.setClass 'focus'
+        else tab.unsetClass 'focus'
```

Comment:
yeah, which is `active` class.

https://github.com/koding/kd/blob/master/lib/components/tabs/tabview.coffee#L240

https://github.com/koding/kd/blob/master/lib/components/tabs/tabhandleview.coffee#L58